Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -568,10 +568,12 @@ void TerminalWidget::paste()
 {
     if (m_ptm_fd == -1)
         return;
-    auto text = GUI::Clipboard::the().data();
-    if (text.is_empty())
+    auto [data, type] = GUI::Clipboard::the().data_and_type();
+    if (type != "text")
         return;
-    int nwritten = write(m_ptm_fd, text.characters(), text.length());
+    if (data.is_empty())
+        return;
```

Comment:
What's wrong with taking the generic code path?

Reply:
You mean without checking if it text first? It would try to paste as string the bitmap contents. I figured out that was not what we wanted while pasting

Reply:
```
Terminal(8): First byte doesn't make sense, bytes: ��������������������������������������������������������������������
USERSPACE(8) ASSERTION FAILED: first_byte_makes_sense
../../AK/Utf8View.cpp:190
[colonel(0:0)]: Terminating due to signal 6

```

Reply:
No, checking that it's text is fine. But why check that it's not empty?

Reply:
Ohh I see. It was introduced in dbf8d6bc1eab19a57e9e6e7f81aa255505e971d4

I didn't want to touch things not strictly related to the PR

Reply:
I see.